### PR TITLE
Update uglifyjs.rst

### PR DESCRIPTION
**Pull request by bouland**
I have to point directly to the node module on Windows with Symfony 2.7.3 and Assetic 2.6.1

### CODE REVIEW

**Comment by xabbuh** [comment]
I am not sure if we should really give a concrete path at all. Instead we could simply use a more generic value like `/path/to/uglifyjs`. What do you think?

**Comment by bouland** [comment]
Indeed the path could be local or absolute. However the comment should explicitly mention the module js NOT the bash script. And notice the difference between JS and CSS one. Although ./node_modules/.bin/uglifyjs point to a bash file whereas ./node_modules/uglify-js/bin/uglifyjs is a Javascript module (in this case the "bin" part is confusing)

File: cookbook/assetic/uglifyjs.rst [hunk 1]
```diff
@@ -75,6 +75,8 @@ your JavaScripts:
                 uglifyjs2:
                     # the path to the uglifyjs executable
                     bin: /usr/local/bin/uglifyjs
+                    # on Windows with a local setup
```
Comment (Nek-) [inline cookbook/assetic/uglifyjs.rst]: This is not specific to windows

Comment (bouland) [inline cookbook/assetic/uglifyjs.rst]: I have no Unix System, Ok i change my PR to reflect your comments

File: cookbook/assetic/uglifyjs.rst [hunk 2]
```diff
@@ -73,8 +73,8 @@ your JavaScripts:
         assetic:
             filters:
                 uglifyjs2:
-                    # the path to the uglifyjs executable
-                    bin: /usr/local/bin/uglifyjs
+                    # the path to the uglifyjs Node module
+                    bin: ./app/Resources/node_modules/uglify-js/bin/uglifyjs
```
Comment (Nek-) [inline cookbook/assetic/uglifyjs.rst]: 1. `node_modules` is like `vendor`, are you ready to put your vendor in this place ?
2. By default `node_module` is not in this place

**Comment by Nek-** [comment]
Btw @xabbuh is probably right.

File: cookbook/assetic/uglifyjs.rst (hunk 2 above)
Comment (bouland) [inline cookbook/assetic/uglifyjs.rst]: Well not !! Mine is at default place (no prefix option) but it was to keep consistency with local install explain previously in this doc.

**Comment by WouterJ** [comment]
I don't think we should merge this. The installation section at the start of this article explains 2 ways: Global and Local. I think we should use the global (recommended) way in the rest of the article.

I hope it is already quite clear that these config values contain the paths to the modules. If not, we should open a new PR making that more clear (e.g. by adding a comment in the code). For now, I'm going to close this one, but feel free to comment if you don't agree.

I thank you for contributing to the Symfony docs, @bouland, I hope to see you back with another PR soon! :christmas_tree:

**Pull request closed without merging**